src/main/java/com/atlassian/bitbucket/jenkins/internal/status/BitbucketBuildStatusFactory.java

```diff
@@ -4,24 +4,25 @@
 import com.atlassian.bitbucket.jenkins.internal.model.BitbucketBuildStatus;
```

Comment:
This class is changed and is slightly different form than this.

Reply:
Yeah, I'm doing a rebase now. These changes are easy. It's the test changes that are killing me. :P